### PR TITLE
generate params document

### PR DESCRIPTION
**Pull request by njzjz**
Usage: `dp docs`

Currently print:
see below

### CODE REVIEW

File: source/train/DescrptSeA.py [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
 from deepmd.RunOptions import global_np_float_precision
 from deepmd.env import op_module
 from deepmd.env import default_tf_session_config
 
 class DescrptSeA ():
-    def __init__ (self, jdata):
+    @argproperty
+    def args(cls):
         args = ClassArg()\
                .add('sel',      list,   must = True) \
```
Comment (njzjz) [inline source/train/DescrptSeA.py:13]:
```suggestion
               .add('sel',      list,   must = True, docs="the maximum possible number of neighbors in the cut-off radius.") \
```
@amcadmus @jameswind The detailed document needs to be filled like this.

Comment (njzjz) [reply source/train/DescrptSeA.py:13]: After all things are done, I will consider generating a web page.

**Comment by njzjz** [comment]
# Params

## descriptor

### loc_frame
|  **sel_a** | list |
|  ----  | ----  |
| *Description:* |  |

|  **sel_r** | list |
|  ----  | ----  |
| *Description:* |  |

|  **rcut** *(optional)* | float |
|  ----  | ----  |
| *Default:* | 6.0 |
| *Description:* |  |

|  **axis_rule** | list |
|  ----  | ----  |
| *Description:* |  |

### se_a
|  **sel** | list |
|  ----  | ----  |
| *Description:* |  |

|  **rcut** *(optional)* | float |
|  ----  | ----  |
| *Default:* | 6.0 |
| *Description:* |  |

|  **rcut_smth** *(optional)* | float |
|  ----  | ----  |
| *Default:* | 5.5 |
| *Description:* |  |

|  **neuron** *(optional)* | list |
|  ----  | ----  |
| *Default:* | [10, 20, 40] |
| *Description:* |  |

|  **axis_neuron** *(optional)* | int |
|  ----  | ----  |
| *Default:* | 4 |
| *Alias:* | n_axis_neuron |
| *Description:* |  |

|  **resnet_dt** *(optional)* | bool |
|  ----  | ----  |
| *Description:* |  |

|  **trainable** *(optional)* | bool |
|  ----  | ----  |
| *Default:* | True |
| *Description:* |  |

|  **seed** *(optional)* | int |
|  ----  | ----  |
| *Description:* |  |

|  **exclude_types** *(optional)* | list |
|  ----  | ----  |
| *Description:* |  |

|  **set_davg_zero** *(optional)* | bool |
|  ----  | ----  |
| *Description:* |  |

|  **activation_function** *(optional)* | str |
|  ----  | ----  |
| *Default:* | tanh |
| *Description:* |  |

|  **precision** *(optional)* | str |
|  ----  | ----  |
| *Default:* | default |
| *Description:* |  |

### se_r
|  **sel** | list |
|  ----  | ----  |
| *Description:* |  |

|  **rcut** *(optional)* | float |
|  ----  | ----  |
| *Default:* | 6.0 |
| *Description:* |  |

|  **rcut_smth** *(optional)* | float |
|  ----  | ----  |
| *Default:* | 5.5 |
| *Description:* |  |

|  **neuron** *(optional)* | list |
|  ----  | ----  |
| *Default:* | [10, 20, 40] |
| *Description:* |  |

|  **resnet_dt** *(optional)* | bool |
|  ----  | ----  |
| *Description:* |  |

|  **trainable** *(optional)* | bool |
|  ----  | ----  |
| *Default:* | True |
| *Description:* |  |

|  **seed** *(optional)* | int |
|  ----  | ----  |
| *Description:* |  |

|  **exclude_types** *(optional)* | list |
|  ----  | ----  |
| *Description:* |  |

|  **set_davg_zero** *(optional)* | bool |
|  ----  | ----  |
| *Description:* |  |

|  **activation_function** *(optional)* | str |
|  ----  | ----  |
| *Default:* | tanh |
| *Description:* |  |

|  **precision** *(optional)* | str |
|  ----  | ----  |
| *Default:* | default |
| *Description:* |  |

### se_ar
|  **a** | dict |
|  ----  | ----  |
| *Description:* |  |

|  **r** | dict |
|  ----  | ----  |
| *Description:* |  |

## fitting_net

### ener
|  **numb_fparam** *(optional)* | int |
|  ----  | ----  |
| *Description:* |  |

|  **numb_aparam** *(optional)* | int |
|  ----  | ----  |
| *Description:* |  |

|  **neuron** *(optional)* | list |
|  ----  | ----  |
| *Default:* | [120, 120, 120] |
| *Alias:* | n_neuron |
| *Description:* |  |

|  **resnet_dt** *(optional)* | bool |
|  ----  | ----  |
| *Default:* | True |
| *Description:* |  |

|  **rcond** *(optional)* | float |
|  ----  | ----  |
| *Default:* | 0.001 |
| *Description:* |  |

|  **seed** *(optional)* | int |
|  ----  | ----  |
| *Description:* |  |

|  **atom_ener** *(optional)* | list |
|  ----  | ----  |
| *Description:* |  |

|  **activation_function** *(optional)* | str |
|  ----  | ----  |
| *Default:* | tanh |
| *Description:* |  |

|  **precision** *(optional)* | str |
|  ----  | ----  |
| *Default:* | default |
| *Description:* |  |

|  **trainable** *(optional)* | list, bool |
|  ----  | ----  |
| *Default:* | True |
| *Description:* |  |

## model

### default
|  **type_map** *(optional)* | list |
|  ----  | ----  |
| *Description:* |  |

|  **data_stat_nbatch** *(optional)* | int |
|  ----  | ----  |
| *Default:* | 10 |
| *Description:* |  |

|  **data_stat_protect** *(optional)* | float |
|  ----  | ----  |
| *Default:* | 0.01 |
| *Description:* |  |

|  **use_srtab** *(optional)* | str |
|  ----  | ----  |
| *Description:* |  |

### wfc, dipole, polar, global_polar
|  **type_map** *(optional)* | list |
|  ----  | ----  |
| *Description:* |  |

|  **data_stat_nbatch** *(optional)* | int |
|  ----  | ----  |
| *Default:* | 10 |
| *Description:* |  |

|  **data_stat_protect** *(optional)* | float |
|  ----  | ----  |
| *Default:* | 0.01 |
| *Description:* |  |

## learning_rate

### exp
|  **decay_steps** *(optional)* | int |
|  ----  | ----  |
| *Description:* |  |

|  **decay_rate** *(optional)* | float |
|  ----  | ----  |
| *Description:* |  |

|  **start_lr** | float |
|  ----  | ----  |
| *Description:* |  |

|  **stop_lr** *(optional)* | float |
|  ----  | ----  |
| *Description:* |  |

## loss

### ener

#### std
|  **start_pref_e** *(optional)* | float |
|  ----  | ----  |
| *Default:* | 0.02 |
| *Description:* |  |

|  **limit_pref_e** *(optional)* | float |
|  ----  | ----  |
| *Default:* | 1.0 |
| *Description:* |  |

|  **start_pref_f** *(optional)* | float |
|  ----  | ----  |
| *Default:* | 1000 |
| *Description:* |  |

|  **limit_pref_f** *(optional)* | float |
|  ----  | ----  |
| *Default:* | 1.0 |
| *Description:* |  |

|  **start_pref_v** *(optional)* | float |
|  ----  | ----  |
| *Description:* |  |

|  **limit_pref_v** *(optional)* | float |
|  ----  | ----  |
| *Description:* |  |

|  **start_pref_ae** *(optional)* | float |
|  ----  | ----  |
| *Description:* |  |

|  **limit_pref_ae** *(optional)* | float |
|  ----  | ----  |
| *Description:* |  |

|  **start_pref_pf** *(optional)* | float |
|  ----  | ----  |
| *Description:* |  |

|  **limit_pref_pf** *(optional)* | float |
|  ----  | ----  |
| *Description:* |  |

|  **relative_f** *(optional)* | float |
|  ----  | ----  |
| *Description:* |  |

#### ener_dipole
|  **start_pref_e** | float |
|  ----  | ----  |
| *Default:* | 0.1 |
| *Description:* |  |

|  **limit_pref_e** | float |
|  ----  | ----  |
| *Default:* | 1.0 |
| *Description:* |  |

|  **start_pref_ed** | float |
|  ----  | ----  |
| *Default:* | 1.0 |
| *Description:* |  |

|  **limit_pref_ed** | float |
|  ----  | ----  |
| *Default:* | 1.0 |
| *Description:* |  |

## training
|  **numb_test** *(optional)* | int |
|  ----  | ----  |
| *Default:* | 1 |
| *Description:* |  |

|  **disp_file** *(optional)* | str |
|  ----  | ----  |
| *Default:* | lcurve.out |
| *Description:* |  |

|  **disp_freq** *(optional)* | int |
|  ----  | ----  |
| *Default:* | 100 |
| *Description:* |  |

|  **save_freq** *(optional)* | int |
|  ----  | ----  |
| *Default:* | 1000 |
| *Description:* |  |

|  **save_ckpt** *(optional)* | str |
|  ----  | ----  |
| *Default:* | model.ckpt |
| *Description:* |  |

|  **display_in_training** *(optional)* | bool |
|  ----  | ----  |
| *Default:* | True |
| *Description:* |  |

|  **timing_in_training** *(optional)* | bool |
|  ----  | ----  |
| *Default:* | True |
| *Description:* |  |

|  **profiling** *(optional)* | bool |
|  ----  | ----  |
| *Description:* |  |

|  **profiling_file** *(optional)* | str |
|  ----  | ----  |
| *Default:* | timeline.json |
| *Description:* |  |

|  **sys_probs** *(optional)* | list |
|  ----  | ----  |
| *Description:* |  |

|  **auto_prob_style** *(optional)* | str |
|  ----  | ----  |
| *Default:* | prob_sys_size |
| *Description:* |  |
